node_modules/gh-orgunit/lib/internal/dao.js

```diff
@@ -760,3 +763,402 @@ var _exportOrgUnitJSON = function(rootId, orgUnits, callback) {
 
     return callback(null, tree);
 };
+
+/* Import */
+
+/**
+ * Import a set of given organisational units, event series and events into an organisational unit
+ *
+ * @param  {MergedTreeData}     mergedTreeData      The merged tree data that contains the tree to import and the data to delete
+ * @param  {Function}           callback            Standard callback function
+ * @param  {Object}             callback.err        An error object, if any
+ */
+var importTree = module.exports.importTree = function(mergedTreeData, callback) {
```

Comment:
The import file and rollover file work on the highest (course) level. At a guess, I think the largest course in Cambridge contains maybe 20k events, 2k series and about 500 org units. 

Postgres has a hard limit of 2^31 number of queries that can be ran, we're probably not going to reach this one.
When a transaction gets committed, postgres needs to be able to process them of course, I assume this is a matter of disk size and available IO. I'll do a couple of imports against the RDS database, but I suspect our datasize is small enough to not pose a problem